### PR TITLE
Update Beautiful Cities of Morrowind

### PR DESCRIPTION
**Pull request by IllusiveMan196**
BCoM load rule enhancements
Vanilla Quest Overhaul load rule
MasterIndexRedux added with load rule

### CODE REVIEW

**Comment by sibir-ine** [comment]
This should be split into at least two PRs, one for the BCoM rules & one for the Vanilla Quest Tweaks rule.

**Comment by IllusiveMan196** [comment]
Separate PR for one line?

**Comment by sibir-ine** [comment]
Yes, PRs should stay focused on a single theme whether it be, e.g., updates for a single mod or Bash Tags or cleaning data for several mods.

File: masterlist.yaml [hunk 1]
```diff
@@ -854,3 +867,4 @@ plugins:
       - 'The Frostmoth Smugglers - Properly Rewarded.ESP'
       - 'Thelas'' Pillows Overhaul.ESP'
       - 'Therana vs Trerayna.ESP'
+    after: [ 'Religions Elaborated.ESP' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: This can be removed from this PR now:

```suggestion
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

File: masterlist.yaml [hunk 2]
```diff
@@ -569,10 +569,19 @@ plugins:
...
+      - 'Better Landscapes Stonewood Pass (MD Edit).esp'
       - 'Better Landscapes Stonewood Pass (RP Edit).esp'
+      - 'DD_Caldera_Expansion v15 + Stonewood Pass Patch.ESP'
+      - 'Fighters_Guild_Questline_Overhaul.ESP'
+      - 'Foyada Mamaea Overhaul.ESP'
       - 'KS_Julan_Ashlander Companion_2.0.esp'
       - 'KS_Julan_Ashlander Companion_3.0_beta.esp'
       - 'Morrowind Anti-Cheese.esp'
+      - 'Trackless Grazeland.ESP'
+      - 'Unofficial Morrowind Official Plugins Patched.ESP'
```
Comment (sibir-ine) [inline masterlist.yaml]: This is already handled by groups:

```suggestion
```

File: masterlist.yaml [hunk 3]
```diff
@@ -829,6 +838,10 @@ plugins:
         display: '[Master Index Plugin](https://www.nexusmods.com/morrowind/mods/42985/) **OR** [Unofficial Morrowind Official Plugins Patched](https://www.nexusmods.com/morrowind/mods/43931/) (**master-index.esp**)'
         condition: 'not active("MasterIndexRedux.esp")'
 
+  - name: 'MasterIndexRedux.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/48977/' ]
+    after: [ 'Beautiful cities of Morrowind.ESP' ]
+
```
Comment (sibir-ine) [inline masterlist.yaml]: LOOT should sort this way by default:

```suggestion
```

**Review by sibir-ine — APPROVED** [review]

**Pull request merged**